Automattic/wp-calypso
# Domains: Show auto renew button if purchased with credits

#### Changes proposed in this Pull Request

This PR enables the auto renew button even if there's no payment method (e.g. the purchase was done with credits). In that case, enabling the auto renew toggle will direct the user to update their payment method.

The auto-renew button should work with any purchase, as long as it's recurring.

#### Testing instructions

* For a domain/mapping purchased with credits, verify that the auto-renew toggle appears, and that it directs you to the payment method screen
* For domains or mappings purchased with a credit card, verify that the auto-renew toggle works as before, so that nothing is broken
